qemu/tests/cfg/blockdev_commit_filter_node_name.cfg

```diff
@@ -0,0 +1,25 @@
+- blockdev_commit_filter_node_name:
+    type = blockdev_commit_filter_node_name
+    virt_test_type = qemu
+    only Linux
+    device_tag = 'data'
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags_data = sn1
+
+    image_size_sn1 = 500M
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+    commit_speed = 10000
+    filter-node-name = 'commit-top'
```

Comment:
it's better to use filter_node_name instead of filter-node-name, which could also be translated to arguments['filter-node-name'] in your script

Reply:
Ok,will update it.